go/vt/sqlparser/ast.go

```diff
@@ -1660,8 +1663,12 @@ func (node *DDL) Format(buf *TrackedBuffer) {
 
 			if node.OptLike != nil {
 				buf.Myprintf("%s%s table%s %v %v", node.Action, temporary, notExists, node.Table, node.OptLike)
-			} else if node.TableSpec != nil {
+			} else if node.TableSpec != nil && node.OptSelect == nil {
```

Comment:
Stuff the `OptSelect ==` and `!=` under `TableSpec != nil`. No need to check `TableSpec` twice for a single variable change.